src/App.tsx

```diff
@@ -1,19 +1,173 @@
 import React from 'react';
-import './App.css';
+import { TodoList } from './components/TodoList';
+
+import { Todo } from './types/Todo';
+
+import './App.scss';
 
 import users from './api/users';
+import todos from './api/todos';
+
+const maxId = todos.sort((a, b) => b.id - a.id)[0].id;
+
+type Props = {};
 
-const App: React.FC = () => {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+type State = {
+  newTodos: Todo[],
+  title: string;
+  userId: number;
+  id: number;
+  titleWarning: boolean;
+  userWarning: boolean;
 };
 
-export default App;
+export class App extends React.Component<Props, State> {
+  state = {
+    newTodos: [...todos],
+    title: '',
+    userId: 0,
+    id: maxId,
```

Comment:
I'd rather not save id inside state, but calculate it dynamically when creating a new todo